### DISCUSSION TITLE
Can I style checkbox label?

### DISCUSSION THREAD

**Discussion by apasov** [quasarframework/quasar#13190] [General - Components / Directives / etc] [answered]
I want to apply custom styles to checkbox label. Font, size, bgcolor, bgimage etc. 
Is this possible directly in Quasar?
Or should I think of some workaround?

**Comment by metalsadman** [comment, marked as answer]
use the default slot, it's for the label.
ie. 
```html
<q-checkbox ...>
  <div class="someClass" style="background: green;">
    Label
  </div>
</q-checkbox>
```
